ts/src/kucoin.ts

```diff
@@ -1096,7 +1096,7 @@ export default class kucoin extends Exchange {
             }
             for (let j = 0; j < chainsLength; j++) {
                 const chain = chains[j];
-                const chainId = this.safeString (chain, 'chain');
+                const chainId = this.safeString (chain, 'chainId');
```

Comment:
@ttodua I don't know how we (me included) let this slip, but we need to be more cautious next time

Reply:
again and again, we need tests. otherwise we might always undeliberately let such slip.
the thing is that kucoin's api https://docs.kucoin.com/#get-currency-detail-recommend mentions `chain`, while actual response is `chainId`.

Reply:
@ttodua ofc we need tests but even more importantly (because tests can't test everything) we need to be careful upon changing these things.